src/DriverFactory.php

```diff
@@ -89,4 +96,24 @@ protected function getDefaultConfig(string $configKey) : array
             'drivers' => [],
         ];
     }
+
+    /**
+     * Registers the annotation loader
+     */
+    private function registerAnnotationLoader() : void
+    {
+        if (self::$isAnnotationLoaderRegistered) {
+            return;
+        }
+
+        /** @psalm-suppress DeprecatedMethod */
+        AnnotationRegistry::registerLoader(
+            /** @psalm-param class-string $className */
+            static function (string $className) {
```

Comment:
This can be `'class_exists'` as a string - no need for the closure

Reply:
Done :+1: